backend/controllers/game-controllers.js

```diff
@@ -247,6 +247,11 @@ function evaluatePlayerAnswer(roomId, playerId, answer) {
       highestPossiblePoints -
       game.roundHistory[currentQuestionIndex].playerRankings.length;
     game.players[playerId].points += pointsToEarn;
+    
+    const roundResult = {
```

Comment:
let's follow something like this for the roundResult object, where we send 2 pieces of data: https://github.com/PlaylistCrafters/CS375Spotify/wiki/Socket-Game-Events#sendroundresults

- a list of players (ordered by points) (we can sort like this: `games[roomId].players.sort((a, b) => b.points - a.points);`
- the players who answered correctly this round (`playerRankings`)